edx_arch_experiments/__init__.py

```diff
@@ -2,4 +2,4 @@
 A plugin to include applications under development by the architecture team at 2U.
 """
 
-__version__ = '1.2.0'
+__version__ = '1.3.0'
```

Comment:
Could you bump this to 2.0.0 to indicate the breaking change?

Reply:
good call, rebased into 002b2ce